api/services/nlp_manager.py

```
@@ -9,8 +9,12 @@
     AutoTokenizer, 
     AutoModelForSequenceClassification, 
     AutoModelForTokenClassification,
+    AutoModelForSeq2SeqLM,
```

Ruff (0.11.9)

`transformers.AutoModelForSeq2SeqLM` imported but unused

Remove unused import

(F401)